src/vs/workbench/contrib/webviewPanel/browser/webviewPanel.contribution.ts

```diff
@@ -27,21 +26,35 @@ import { IWebviewWorkbenchService, WebviewEditorService } from './webviewWorkben
 	localize('webview.editor.label', "webview editor")),
 	[new SyncDescriptor(WebviewInput)]);
 
-class WebviewPanelContribution implements IWorkbenchContribution {
+class WebviewPanelContribution extends Disposable implements IWorkbenchContribution {
+
+	private readonly _disposables = new DisposableStore();
```

Comment:
You can  use `_register(new DisposableStore())` to avoid having to reimplement `dispose` later on